admin/app/adapters/application.js

```diff
@@ -1,17 +1,29 @@
+import { computed } from '@ember/object';
 import JSONAPIAdapter from '@ember-data/adapter/json-api';
 import DataAdapterMixin from 'ember-simple-auth/mixins/data-adapter-mixin';
+import { task } from 'ember-concurrency';
 import ENV from 'pix-admin/config/environment';
 
-export default class Application extends JSONAPIAdapter.extend(DataAdapterMixin) {
+export default class ApplicationAdapter extends JSONAPIAdapter.extend(DataAdapterMixin) {
 
   host = ENV.APP.API_HOST;
   namespace = 'api';
 
+  @computed('session.data.authenticated.access_token')
   get headers() {
     const headers = {};
     if (this.session.isAuthenticated) {
       headers['Authorization'] = `Bearer ${this.session.data.authenticated.access_token}`;
     }
     return headers;
   }
+
+  @(task(function*(ajaxFn,...ajaxArgs) {
+    return yield ajaxFn(...ajaxArgs);
+  }).enqueue().maxConcurrency(ENV.APP.MAX_CONCURRENT_AJAX_CALLS))
+  _ajaxThrottleTask;
+
+  ajax() {
+    return this._ajaxThrottleTask.perform(super.ajax.bind(this), ...arguments);
+  }
```

Comment:
Je crois que j'ai compris pourquoi je continue à voir plus de requêtes simultanées que ce que j'ai configuré : ici c'est l'instance de l'`adapter` qui trace les requêtes simultanées. Or il n'est pas garanti par Ember qu'il n'y aura qu'une seule instance d'`adapter`. Pour les cas où c'est une classe dérivée (ex. `adapters/challenge.js`) c'est évident que ça ne peut pas être la même instance, mais même pour le cas où c'est la même classe qui est utilisée ça n'est pas garanti (https://discuss.emberjs.com/t/how-many-instances-of-adapter-application-may-there-be/9038).

Du coup on se retrouve avec `MAX_CONCURRENT_AJAX_CALLS` appels simultanés… par instance.

Pour contourner ça il faut trouver un moyen de mettre ce "limitateur" dans un objet dont on est sûr qu'il ne sera instancié qu'une fois pour toute l'application. J'ai l'impression qu'utiliser un service Ember enregistré comme _singleton_ serait correct : https://guides.emberjs.com/v3.17.0/applications/dependency-injection/#toc_registering-singletons-vs-non-singletons